js/stdlib/ts/ETH_like.ts

```diff
@@ -1176,6 +1182,13 @@ function setCustomHttpEventHandler() {
   console.warn(`setCustomHttpEventHandler is not supported on this connector`);
 }
 
+const doAppOptIn = async (_ctc: ContractInfo): Promise<void> => {
+  return;
+};
+const accountAppOptedIn = async (_acc: Account, _ctc: ContractInfo): Promise<boolean> => {
```

Comment:
This function should be included in `interfaces.ts` as part of the stdlib (and it needs to take `Account | Address`